css-four-column-layout/styles.css

```diff
@@ -13,12 +9,17 @@
 
 .column-one-fourth {
   flex-basis: 25%;
+  background-color: rgb(247 248 250);
+  padding-left: 50px;
+  padding-right: 50px;
+  padding-bottom: 13px;
```

Comment:
Only `flex-basis` should be applied to `column-` classes. Make another class that you can add to the element in addition to the `column-one-fourth` class.